### PR TITLE
Restructuring TARDIS Documentation (Issue #948)

### LINKED ISSUE #948: TARDIS documentation restructure (I of many)
TARDIS is not only compatible with Python 2.7
typo in recommend on installation
update the link to tardis_env3. yml

change the running tardis to tardis configuration file
move formal integral to configuration

move stella to helper scripts

### PR DESCRIPTION
**Pull request by shreyansjain012**
This PR addresses some of the changes needed in TARDIS documentation.
## Description

- `docs/installation.rst`: Rephrased the content  
- `docs/running/index.rst`: Changed *Running TARDIS* to *TARDIS Configuration File* 
- `docs/running/interaction/integrator.ipynb` → `docs/configuration/integrator.ipynb`
- `docs/configuration/`: Added a link to `integrator`
- The first link of the notebook given [here](https://tardis-sn.github.io/tardis/research/code_comparison/toy_models/reading%20blondin%20toymodel.html) was dead because of spacing between the name. Renamed `reading blondin toymodel.ipynb` to `reading_blondin_toymodel.ipynb`
- Many files still contains python2 syntax, update files to use python3 syntax
## Motivation and Context

- Good documentation is always required for a large project.

- It partially fixes #948
## How Has This Been Tested?

- Ran documentation on a local server using `make html` and verified that each change is working.

**OS - Ubuntu 18.04**

## Screenshots (if appropriate):
**Outdated syntax**
syntax_error

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by shreyansjain012** [comment]
Please let me know if this PR needs some changes.

**Comment by jaladh-singhal** [comment]
@shreyansjain012 Please be more descriptive in your PR titles and description.

I think @wkerzendorf wants you to move [this formal integral page](https://tardis-sn.github.io/tardis/running/interaction/integrator.html?highlight=formal%20integral) under [configuration section](https://tardis-sn.github.io/tardis/configuration/index.html?highlight=configuration).

Additionally, have a look at past gitter chat where [he explained about it](https://github.com/tardis-sn/tardis/issues/948#issuecomment-587751540)

**Comment by shreyansjain012** [comment]
> @shreyansjain012 Please be more descriptive in your PR titles and description.
> 
> I think @wkerzendorf wants you to move [this formal integral page](https://tardis-sn.github.io/tardis/running/interaction/integrator.html?highlight=formal%20integral) under [configuration section](https://tardis-sn.github.io/tardis/configuration/index.html?highlight=configuration).
> 
> Additionally, have a look at past gitter chat where [he explained about it](https://github.com/tardis-sn/tardis/issues/948#issuecomment-587751540)

I went to see previous chats but didn't find any mentions regarding this issue. 
Care to explain it again? 😅

**Comment by jaladh-singhal** [comment]
Wait until @wkerzendorf can reply you back. Meanwhile, go through documentation and identify if there is something that needs to be restructured as per you?

File: docs/running/interaction/hdf_writer.ipynb [hunk 1]
```diff
@@ -67,9 +63,7 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```
Comment (jaladh-singhal) [inline docs/running/interaction/hdf_writer.ipynb]: Does this automatically becomes empty in Python 3 or did you do it for any specific purpose?

Comment (shreyansjain012) [reply docs/running/interaction/hdf_writer.ipynb]: I didn't change it, let me look into it

Comment (shreyansjain012) [reply docs/running/interaction/hdf_writer.ipynb]: Actually this automatically becomes empty when for python 3

Comment (jaladh-singhal) [reply docs/running/interaction/hdf_writer.ipynb]: So you practically ran these both notebooks in python3 (latest tardis) environment?

Comment (shreyansjain012) [reply docs/running/interaction/hdf_writer.ipynb]: Yes

**Comment by jordi5** [comment]
Hi! Please rebase this branch and check that there are no conflicts with the new documentation edits introduced in #1158.

**Comment by wkerzendorf** [comment]
closing as outdated

**Pull request closed without merging**